docs/src/components/home/ProductsSwitcher.tsx

```diff
@@ -100,7 +100,7 @@ const ProductsSwitcher = ({
       aria-label="Go to core components page"
       icon={<IconImage name="product-core" />}
       name="MUI Core"
-      description="Ready-to-use foundational components, free forever."
+      description="Foundational components for shipping features faster. Includes Material UI."
```

Comment:
Should we extend to clarify that it includes Material UI but much more other things apart from it. Not sure :)

Reply:
The idea here was to name Material UI specifically since it's something that probably the majority of users will be looking for when they land on the homepage. We can go into more detail about Core's products on the planned mui.com/products page.